frontend/src/pages/node/AssociationsTable.vue

```diff
@@ -65,6 +90,15 @@
       <span v-else class="empty">No info</span>
     </template>
 
+    <template #has_evidence="{ row }">
+      <AppLink
+        v-for="(source, index) in row.has_evidence_links"
+        :key="index"
+        :to="source.url || ''"
+        >{{ source.id }}</AppLink
+      >
+    </template>
+
```

Comment:
how to display a has_evidence cell (borrowed from SectionAssociationDetails)

Reply:
You likely shouldn't be using `index` as a key here. If the table gets resorted, this may not be re-rendered. See the [Vue documentation](https://vuejs.org/api/built-in-special-attributes#key), and also #892 for a fix to a similar bug.

The key should uniquely identify what objcet this is rendering. I'm not totally familiar with this data structure, but I'm guessing that would be `source.id`.

For example, if I had the set of rows:
```
[
  { source: { id: 'a' }}, // index is 0
  { source: { id: 'b' }}, // index is 1
  { source: { id: 'c' }}, // index is 2
]
```

Vue would render if fine initially. But if I resorted those rows:

```
[
  { source: { id: 'c' }}, // index is 0
  { source: { id: 'b' }}, // index is 1
  { source: { id: 'a' }}, // index is 2
]
```

Vue would not know that it would need to re-render those rows, since the key of the elements never changes-- the first element always has index 0, the second index 1, and so on.

Reply:
fixed! thanks for catching that!